topics/Transportation_networks.md

```diff
@@ -76,17 +76,27 @@ What is coded? A protocol should be developed to ensure consistency throughout t
 
 **Standards**
 
-There are various emerging standards for transportation networks. Standard network definitions make it easier to share networks between organizations, software programs, and people. A few relevant standards include:
+There are various emerging standards for transportation networks. A few relevant standards include:
 
 -   OpenStreetMap - <http://wiki.openstreetmap.org/wiki/Main_Page>
 -   General Transit Feed Specification (GTFS) - <https://developers.google.com/transit/gtfs>
 -   Model Free - <http://modelfree.org>
 -   UNETRANS - <http://ncrst.digitalgeographic.com/research/unetrans/first.html>
--   TIGER/Line - <http://www.census.gov/geo/maps-data/data/tiger.html>
+-   TIGER/Line - <https://tigerweb.geo.census.gov/tigerwebmain/TIGERweb_geography.html>
```

Comment:
I'm concerned about calling many of these files "network standards" when they are in fact "spatial information standards." OSM, GTFS, and TIGER all take substantial work to turn into an actual routable network.